Tactical/Overhead.cpp

```diff
@@ -11092,6 +11034,105 @@ void HandleTurncoatAttempt( SOLDIERTYPE* pSoldier )
 	}
 }
 
+void EscapeTimerCallback()
+{
+    const bool chanceToEscape = Chance(75);

```

Comment:
this can be static

Reply:
Wouldn't that make it be called only once, and then the chance is set in stone for every subsequent surrender during the whole duration of the program?

Reply:
yes, I was wrong there. my bad.